src/main/scala/tech/sourced/gemini/cmd/HashSparkApp.scala

```diff
@@ -136,6 +140,16 @@ object HashSparkApp extends App with Logging {
       log.info("Checking DB schema")
       CassandraConnector(spark.sparkContext).withSessionDo { cassandra =>
         gemini.applySchema(cassandra)
+
+        if (config.replace) {
+          gemini.cleanDB(cassandra, config.mode)
+        }
```

Comment:
Maybe better using `else if` in order to avoid checking for emptiness if just cleaned?
```suggestion
        } else if (!gemini.isDBEmpty(cassandra, config.mode)) {
          println("Database keyspace is not empty! Hashing may produce wrong results. " +
            "Please choose another keyspace or pass the --replace option")
          System.exit(2)
        }
```

Reply:
thanks! fixed